tsconfig.json

```diff
@@ -8,15 +8,16 @@
         // Search under node_modules for non-relative imports.
         "moduleResolution": "node",
         // Process & infer types from .js files.
-        // "allowJs": true,
+        "allowJs": true,
```

Comment:
These changes allow the offline service package to be JS for now. When possible, it will be converted into Typescript

Reply:
Something in this file is causing an error in the CI though!